Recruit-Training/Advanced-Recruit-Training/src/LogFileReader.cpp

```diff
@@ -18,6 +18,8 @@ LogFileReader::LogFileReader()
 
 LogFileReader::~LogFileReader()
 {
+
+
```

Comment:
Double check your PR for changes like this, since it doesn't do anything. Changes that don't do anything adds overhead for reviewers and makes it tougher to review.